pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -773,6 +773,6 @@ stdenv.mkDerivation (finalAttrs: {
     # https://github.com/systemd/systemd/issues/20600#issuecomment-912338965
     broken = stdenv.hostPlatform.isStatic;
     priority = 10;
-    maintainers = with maintainers; [ flokli kloenk mic92 ];
```

Comment:
Probably I will still do some upstream work in a once in a while, but I am getting more pings from maintainer team than I can handle w.r.t. to my other projects.